src/main/java/mat/server/MeasureLibraryServiceImpl.java

```diff
@@ -1138,7 +1139,7 @@ public ManageMeasureSearchModel getAllRecentMeasureForUser(String userId) {
             lockedUserInfo.setLastName(measure.getLockedUser().getLastName());
             detail.setLockedUserInfo(lockedUserInfo);
         }
-
+        detail.setValidatable(MeasureDetailsUtil.isValidatable(measure.getReleaseVersion(), measure.isDraft(), detail.isDraftable(), measure.getMeasureModel()));
```

Comment:
I would pass in measure object to `isValidatable(measure)`

Reply:
Yeah. I agree. It makes sense to mass measure object. Also, the pattern for other similar methods MeasureDetailsUtil.get().isValidatable

Reply:
But one of the parameter we need is isDraftable, which is not present in measure Entity. isDraftable is decided based on isDraft(). But @serhii-ilin you told the other day that its not a good practice to use isDraft() to decide if it is Draftable. 

isValidatable() is being called for recent activity grid and measure grid, though both the calls are from same class, they use 2 different Models(measure & dto).
`detail.setValidatable(MeasureDetailsUtil.isValidatable(dto.getReleaseVersion(), dto.getQdmVersion(), dto.isDraft(), dto.isDraftable(), dto.getMeasureModel()));
`

Reply:
I think there is a misunderstanding... It's not what I meant. Will ping you on that